packages/editable-html/src/editor.jsx

```diff
@@ -261,6 +262,13 @@ export class Editor extends React.Component {
     if (!isEqual(nextProps.languageCharactersProps, this.props.languageCharactersProps)) {
       this.handlePlugins(nextProps);
     }
+
+    if (!nextProps.value.document.equals(this.props.value.document)) {
+      this.setState({
+        focus: false,
+        value: nextProps.value,
+      });
+    }
```

Comment:
Please don't forget to mention in the ticket that this change needs to be well tested

Reply:
Of course, giving details about what could go wrong